README.md

```diff
@@ -0,0 +1,77 @@
+## Demo Feedback Guidelines/Requests
+Please give feedback within *5 business days* by 14apr2023 1700 PST<br>
+Please use review types:  Grammar or Typo (G), Unclear (U), Technical Accuracy (T), or Other (O)
+
+G: This doesn’t flow, instead, what about this:<br>
+U: When I read this I think “xyz”, which is confusing to what you mean, can you help clarify what you meant?<br>
+T: This doesn’t reflect the response we should get, instead we should see “xyz”<br>
+O: This was mentioned earlier, can we combine?
+
+# WIP Learn Temporal Python SDK v2
+This project is intended to show my process through learning [Temporal](http://temporal.io/) and is not meant to showcase production-level best practices.
+
+The program is a simple implementation of a five card draw poker game using the [Temporal Python SDK](https://github.com/temporalio/sdk-python). The main components of the application are:
+
+`main.py`: Starts the Temporal client, connects to the server, and initializes a worker to execute the `PokerWorkflow` and `HandRankingWorkflow` workflows. It then executes the `PokerWorkflow` with a seed value.
+
+`poker_workflow.py`: Defines the `PokerWorkflow`, which simulates a poker game. It creates and shuffles a deck of cards, deals hands to players, and prints their hands. It then evaluates the hands using the `HandRankingWorkflow` and determines the winner.
+
+`hand_ranking.py`: Contains utility functions to rank poker hands and draw cards.
+
+`hand_ranking_workflow.py`: Defines the `HandRankingWorkflow`, which takes a poker hand as input and returns its rank.
+
+`deck_utils.py`: Contains utility functions to create, shuffle, and deal cards from a deck.
+
+`game_state.py`: Defines a `GameState` class that represents the state of the game, including the deck of cards and player hands.
+
+`card.py`: Defines Card, Suit, and Rank classes, which are used to represent playing cards in the application.
+
+Learn Temporal Python SDK v1 is [over here](https://github.com/rainleander/learn-temporal-pythonSDK).
```

Comment:
O: If this is a multi-step journey, I think it's clearer to put all steps in the same repo.

Reply:
I disagree; we're keeping the versions separate on purpose, for now. If we decide to merge them into temporal's official repos, then we can discuss merging the projects.